index.bs

```diff
@@ -5071,7 +5072,7 @@ object=] for a credential. Its format is shown in <a href="#table-attestedCreden
             <th>Description</th>
         </tr>
         <tr>
-            <td><dfn>aaguid</dfn></td>
+            <td>AAGUID</td>
```

Comment:
This breaks quite a lot of autolink references to `[=aaguid=]`, often phrased as `the [=aaguid=] in the [=attested credential data=]` or similar. But since this table is scoped with `dfn-for="authData/attestedCredentialData"`, we can keep both `<dfn>`s. We'll need to disambiguate the links as either `[=authData/attestedCredentialData/aaguid=]` or `[=/AAGUID=]`, though.

I test-ran this myself to see that it works, so I'll submit a meta-PR with this suggestion.

Reply:
Meta-PR here: https://github.com/w3c/webauthn/pull/1985